product_docs/docs/epas/15/epas_compat_sql/65a_merge.mdx

```diff
@@ -11,22 +11,19 @@ title: "MERGE"
 ```sql
 MERGE INTO target_table_name [ target_alias ]
 USING data_source [ source_alias ] ON ( join_condition )
-WHEN MATCHED THEN
-merge_update_clause merge_insert_clause
-
 -- merge_update_clause
 WHEN MATCHED THEN 
-   UPDATE SET column = value, ... WHERE where_condition
-   DELETE WHERE where_condition
+   UPDATE SET column = value, ... [ WHERE where_condition ]
+   [ DELETE [ WHERE where_condition ] ]
 
 -- merge_insert_clause
 WHEN NOT MATCHED THEN 
-   INSERT [(col_list)] VALUES (val_list) WHERE where_condition 
+   INSERT [(col_list)] VALUES (val_list) [ WHERE where_condition ] 
 ```
 
 ## Description
 
-`MERGE` allows you to select rows from one or more sources for update or insertion into a table or view. You can specify conditions to determine whether to UPDATE or INSERT into the target table or view. You can also do conditional UPDATE and INSERT using the WHERE clause in the MERGE statement.
+`MERGE` allows you to select rows from one or more sources for update or insertion into a table. You specify join condition to determine whether to UPDATE or INSERT into the target table. Optionally you specify conditional UPDATE and INSERT using the WHERE clause in the MERGE statement.
```

Comment:
Use lowercase when using SQL statement names as verbs instead of nouns. (you might want to check with Betsy about this)
```suggestion
`MERGE` allows you to select rows from one or more sources for update or insertion into a table. You specify the JOIN condition to determine whether to update or insert into the target table. Optionally, you specify conditional UPDATE and INSERT statements using the WHERE clause in the MERGE statement.
```

Reply:
Applied changes in separate commit